assignment.rb

```diff
@@ -0,0 +1,100 @@
+require "csv"
+require "./user.rb"
+
+### CSV set-up
+current_users = []
+csv = CSV.foreach('./users.csv', headers: true, header_converters: :symbol) { |row|
```

Comment:
If you have a block that goes on to more than 1 line, the typical ruby way of starting/ending it is with `do` and `end` instead of `{` and `}`